src/Workspaces/VisualBasic/Portable/Recommendations/VisualBasicRecommendationServiceRunner.vb

```diff
@@ -181,7 +191,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Recommendations
                     End If
                 End If
 
-                Return FilterToValidAccessibleSymbols(symbols)
+                Return symbols
+            End Function
+
+            Private Function IsInterfaceImplementsErrorBlock() As Boolean
+                Dim possibleTypeBlock = _context.TargetToken.Parent?.FirstAncestorOrSelf(Of TypeBlockSyntax)()
+                If possibleTypeBlock IsNot Nothing Then
+                    Dim typeSymbol As ISymbol = _context.SemanticModel.GetDeclaredSymbol(possibleTypeBlock, _cancellationToken)
+                    If typeSymbol.IsInterfaceType() Then
+                        Dim token = _context.TargetToken
+                        If token.IsChildToken(Of ImplementsStatementSyntax)(Function(n) n.ImplementsKeyword) Then
+                            Return True
+                        End If
+                        Return token.IsChildToken(Of QualifiedNameSyntax)(Function(n) n.DotToken) AndAlso
+                               token.Parent?.FirstAncestorOrSelf(Of ImplementsStatementSyntax) IsNot Nothing
+                    End If
+                End If
+                Return False
```

Comment:
I'm not sure what's going on here

Reply:
i guess i'm not really understanding.  if no items were being shown here before, why did they start getting shown?